2025/2025-07-08.md

```diff
@@ -0,0 +1,101 @@
+    - JM: Would be good strategically to add a couple of items in many different languages.
+        - So that beginners have something to do and look at, and for others will make it easier to get started, it's easier to expand something that start from nothing.
+        - RC has a few that could be done in C / has some drafts kicking around on a laptop that were never finalised.
+        - RC: if you want to contribute / review changes in advance of the workshop, please do so via [GitHub](https://github.com/sig-rpc/sig-rpc.github.io/issues))
+    - Workshop agenda
+        - Introduction to the SIG slides (~15 mins)
+        - Work on contributions for (~60 mins)
+            - Paper should be available if as an alternative to laptops, will require digitising afterwards
+        - Short wrap up. Will need to adapt the timing on the day (depends on attendance & participation).
+        - RC will put some slides together
+            - will be able to adapt some slides from a short talk being delivered on 2025-07-10 at the N8 CIR annual RSE meetup in York
+    - Changes to the submission text (abstract)
+        - Some formatting tweaks (fonts, bold list item)
+        - If anyone has other suggested changes let RC know.
+    - Stickers or pens etc.
+        - RC is not a designer - does anyone want to make a logo etc? otherwise it will likely be the text logo on the website in a hexagon
+    - Conference event support
+        - RF may no longer be available to attend RSEcon, would anyone attending be willing to help at the workshop?
+            - JM happy in principle, but unsure on availability with other conference commitments
+            - FS not attending
+            - PTH: RSEcon volunteers may be able to support with some facilitation (e.g. of online)
+- AGM in October
+    - Should schedule this so that it can be advertised in RSEcon.
+    - This will likely be the next meeting, unless another one is needed/wanted in advance of the workshop at RSEcon.
+- AoB
+    - JM promoted SIG-RPC and the PP&O course at RSLondon on 2025-07-08
+        - Had a suggestion from a HPC attendee to try not to make HPC sound too intimidating
+        - HPC SIG would be in principal be interested in a collaborative event
+        - What's the dividing line between RPC and HPC - not sure we have a good 1 sentence answer.
+            - RC: Stuff running on laptops etc. command line which some researchers are intimidated by. Very much single machine, serial, etc. Interactive workloads which may not fit traditional HPC well. Can direct people to HPC as follow on steps.  "pre-HPC"
+            - FS: "Before HPC on Single PC"
+        - WH: presented an R and python package which makes using HPC appear challenging without the user needing to be exposed to it (immediately after central IT presented on traditional hpc/bash training). Some feedback that making HPC easy to use should *not* be made easier.
```

Comment:
I found this bullet point confusingly worded; but unfortunately I don’t remember enough context to reword/expand it. Since we’re hoping to collaborate with HPC RSE SIG, it’d be good to be clear and avoid misunderstandings here.

Reply:
I'll email Wes to have a look.

Reply:
```suggestion
        - WH: presented an R and python package which aims to make using HPC like a simple function call for data scientists. (immediately after central IT presented on traditional hpc/bash training across departments). Some ,indirectly received, feedback from those attending his talk suggested making HPC easy is *not* something we should be aiming to do (e.g. HPC is a privileged resource, they should be learning bash etc).
```

Improved summary from Wes, with a few additional clarifications based on a longer description I was sent.

Reply:
Much clearer, thanks!